Integrations/APIVoid/APIVoid.py

```diff
@@ -0,0 +1,771 @@
+        api_params = {'key': self.apikey, 'url': url}
+        response = super()._http_request(method='GET', url_suffix=suffix, params=api_params)
+        return (response)
+
+    def url_to_html(self, url):
+
+        suffix = '/urltohtml/v1/pay-as-you-go/'
+        api_params = {'key': self.apikey, 'url': url}
+        response = super()._http_request(method='GET', url_suffix=suffix, params=api_params)
+        return (response)
+
+    def site_trust(self, host):
+
+        suffix = '/sitetrust/v1/pay-as-you-go/'
+        api_params = {'key': self.apikey, 'host': host}
+        response = super()._http_request(method='GET', url_suffix=suffix, params=api_params)
+        return (response)
+
+
+def test_module(client):
+
+    result = client.test()
+
+    if (result['success']):
+        return 'ok'
+    else:
+        return ('Test Failed: ' + str(result))
+
+
+def ip_command(client, args, reputation_only):
+
+    ip = args.get('ip')
+    raw_response = client.check_ip(ip)
+
+    if 'error' in raw_response:
+        return_error("Command Failed: " + str(raw_response['error']))
+
+    report = raw_response.get('data', {}).get('report', None)
+    if report:
+        apivoid = dict()
+        for k, v in report.items():
+            apivoid[k] = v
+        engines = apivoid.get('blacklists', {}).get('engines_count', 0)
+        detections = apivoid.get('blacklists', {}).get('detections', 0)
+
+        # IP Information
+        information = apivoid.get('information', {})
+        ip = dict()
+        ip['Address'] = apivoid['ip']
+        ip['Hostname'] = information.get('reverse_dns', None)
+        geo = dict()
+        lat = information.get('latitude', None)
+        lng = information.get('longitude', None)
+        geo['Location'] = f'{lat}:{lng}' if lat and lng else None
+        geo['Country'] = information.get('country_name', None)
+        geo['Description'] = information.get('isp', None)
+        ip['Geo'] = geo
+        ip['DetectionEngines'] = engines
+        ip['PositiveDetections'] = detections
+
+        # DBot Information
+        if detections and engines:
+            detectionRate = (detections / engines) * 100
+        else:
+            detectionRate = 0
+        dbotScore = 0
+        if detectionRate < client.good:
+            dbotScore = 1
+        if engines == 0:
+            dbotScore = 0
+        if detectionRate > client.suspicious:
+            dbotScore = 2
+        if detectionRate > client.bad:
+            dbotScore = 3
+        if (client.malicious == "suspicious" and dbotScore >= 2) or (client.malicious == "bad" and dbotScore == 3):
+            ip['Malicious'] = dict()
+            ip['Malicious']['Vendor'] = 'apivoid'
+            ip['Malicious']['Description'] = f"Detection rate of {ip['PositiveDetections']}/{ip['DetectionEngines']}"
+
+        ec = {
+            'IP(val.ip && val.ip == obj.ip)': ip,
+            'DBotScore': {
+                'Score': dbotScore,
+                'Vendor': 'URL Void',
```

Comment:
vendor should match the integration ID. relevant for all places.
```suggestion
                'Vendor': 'apivoid',
```

Reply:
I included all of the outputs pffered by APIVoid. There are a LOT, but, they are not necessary always completed. There are far too many to fill in the descriptions for. Happy to fill them in, but, it WILL take a while.